Makefile

```diff
@@ -258,10 +259,8 @@ bundle/redhat: bundle
 
 # e2e
 .PHONY: e2e-kind
-e2e-kind:
-ifeq (,$(shell kind get clusters ))
-	$(KIND) --kubeconfig="${KUBECONFIG}" create cluster --image=kindest/node:v$(ENVTEST_K8S_VERSION) --config tests/e2e/kind.yaml
-endif
+e2e-kind: kind
+	@KIND_NODE_VERSION="$(ENVTEST_K8S_VERSION)" RECREATE_CLUSTER="no" hack/kind/start-kind.sh
```

Comment:
This change breaks usage of e2e matrix. - You always install the same version of kind.